scripts/run_dependencies

```diff
@@ -4,32 +4,43 @@ set -ex
 HERE=$(realpath "$(dirname $0)")
 . $HERE/common
 
-NETWORK=beebop_nw
-VOLUME=beebop-storage
-NAME_REDIS=beebop-redis
-NAME_API=beebop-py-api
-NAME_WORKER=beebop-py-worker
-PORT=5000
 
-docker volume create $VOLUME
-docker run --rm -v $VOLUME:/beebop/storage \
+while [ $# -gt 0 ]; do
+  case "$1" in
+    -mount)
+      MOUNT="$2"
+      shift 2
+      ;;
+    *)
+      shift
+      ;;
+  esac
+done
```

Comment:
this is shared logic with run _test to find the mount arg - could pull this out into a shared script - or just use a positional arg if it's easier! But it's quite nice and clear to have a named arg.